app/pb_kits/playbook/pb_background/_background.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= content_tag(:tag,
```

Comment:
Instead of `:tag`, I think you'll need to call the `tag` prop on the view object. So `object.tag`.